### DISCUSSION TITLE
Issue with Root & SSH- Network Error: Connection Refused

### DISCUSSION THREAD

**Discussion by S3cr1t** [Guilouz/Creality-Helper-Script-Wiki#462] [Issues]
Printer: K1 Max

So far:

- Reset Default Settings through touchpad
- Loaded older Creality Firmware, then reloaded Root Firmware
- Allowed Root Access through the Printer touchpad
- Attempted this with different Firmware versions.
- Checked Firewall Settings
- Checked Router settings

Issue:
-Can't connect to the printer via SSH, through Orca Slicer, or Web Interface

Error:
-Network Error: Connection Refused

**Comment by halTWO3** [comment]
You probably done this already, but  by any chance, did you verify the ip of the printer via display screen? Just maybe the router assigned a new one, since you mentioned you reloaded different firmwares.
